### ISSUE TITLE
[next branch]: rprompt is always on second line

### ISSUE THREAD

**Issue by billimek** [bhilburn/powerlevel9k#841]
See [this conversation](https://gitter.im/bhilburn/powerlevel9k?at=5afd9fc1a2d951363349cf9d) for some more context.

On the `next` branch, the rprompt is being displayed on the newline regardless of the POWERLEVEL9K_RPROMPT_ON_NEWLINE=false setting. This is not the case on `master` branch.

This is an example configuration where this situation is observed:

```
POWERLEVEL9K_MODE='nerdfont-complete'
POWERLEVEL9K_PROMPT_ON_NEWLINE=true
POWERLEVEL9K_RPROMPT_ON_NEWLINE=false
POWERLEVEL9K_MULTILINE_FIRST_PROMPT_PREFIX=""
POWERLEVEL9K_MULTILINE_LAST_PROMPT_PREFIX="❯ "
POWERLEVEL9K_CUSTOM_CF_STRING="cf-get-info"
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(context dir rbenv vcs custom_cf_string)
```

@dritter suggests that this was caused by the following change: https://github.com/bhilburn/powerlevel9k/commit/9a895bc7e4c925658a9c57681437000445f399ed#diff-ea5a405f0331722fc10a832209ffe20bR1711

**Comment by notrev** [comment]
This PR might fix your issue: #843

**Comment by billimek** [comment]
I see that this PR is going into master branch.  But wasn't experiencing this on the master branch, just the next branch.
